src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objStyles = {};
+
+  sourceString.split(';').map((style) => {
+    const arrStyle = style.split(':').map((el) => el.trim());
+
+    objStyles[arrStyle[0]] = arrStyle[1];
```

Comment:
This line assumes that `arrStyle[0]` is always a valid key, but there is no check for empty keys. You should add a condition to ensure that only valid CSS property names are used as keys in the `objStyles` object.